pages/osx/netstat.md

```diff
@@ -16,7 +16,7 @@
 
 - Display PID and program names for a specific port:
```

Comment:
Wait there is an inconsistency here. The description says port, but the example says protocol. Which is it ? Does anyone know ? @cdminigun

Reply:
From Mac man pages, it looks like it is protocol. But I don't have my Macbook with me to test. :thinking_face:

Reply:
Is `netstat` on macOS the same as on Linux?

If so, then `-p` does not, in fact, take a parameter. It causes it to display the program names in addition to the information displayed already.

```
$ netstat --version
net-tools 2.10-alpha
Fred Baumgarten, Alan Cox, Bernd Eckenfels, Phil Blundell, Tuan Hoang, Brian Micek and others
+NEW_ADDRT +RTF_IRTT +RTF_REJECT +FW_MASQUERADE -I18N
AF: (inet) +UNIX +INET +INET6 +IPX +AX25 +NETROM +X25 +ATALK -ECONET +ROSE -BLUETOOTH
HW:  +ETHER +ARC +SLIP +PPP +TUNNEL -TR +AX25 +NETROM +X25 +FR +ROSE +ASH +SIT +FDDI +HIPPI +HDLC/LAPB +EUI64 
```

Reply:
No, it's different to Linux afaik. Forgot to check my Mac yesterday _sigh_

Reply:
Ah, I see.

Reply:
-p protocol
           Show statistics about protocol, which is either a well-known name for a proto-
           col or an alias for it.  Some protocol names and aliases are listed in the file
           /etc/protocols.  The special protocol name ``bdg'' is used to show bridging
           statistics.  A null response typically means that there are no interesting num-
           bers to report.  The program will complain if protocol is unknown or if there
           is no statistics routine for it.

For instance, netstat -p icmp

Reply:
I can confirm that on Mac it is protocol. 👍
```
netstat [-AaLlnIN] [-f address _f mily I -p protocol]
netstat [-gilns] E-f address _f HilY]
netstat -i I -I interface [ it: [--abdgRtS]
netstat -s [-s] [-f address family -p protocol] [--w wait]
netstat -i I -I interface -s [-f address family I -p protocol]
netstat -m [-m]
netstat -r [-Aaln] [-f address_family]
netstat -rs [-is] 
```

Reply:
Thanks! Sorry for the confusion, mind must have been on ports at the time.

Reply:
No problem @cdminigun!